tutorials/monitoring-setup/index.md

```diff
@@ -101,7 +110,7 @@ The listing for the ServiceControl instance includes a URL link. This URL will b
 NOTE: Creating the ServiceControl instance will also create the audit and error queues if they did not already exist. By default these are called _audit_ and _error_ respectively.
 
 
-### Create Monitoring Instance
+### Create ServiceControl Monitoring Instance
```

Comment:
For me mentioning ServiceControl here is unnecessary clutter in the title but that might be just me

Reply:
In the added TOC the item would be  "Create Monitoring Instance" which is not really clear. Everywhere we call it ServiceControl Monitoring AFAIK.